dotnet/src/Connectors/Connectors.UnitTests/OpenAI/FunctionCalling/KernelFunctionMetadataExtensionsTests.cs

```diff
@@ -174,6 +174,31 @@ public void ItCanCreateValidOpenAIFunctionManual()
         );
     }
 
+    [Fact]
+    public void ItCanCreateValidOpenAIFunctionManualForPrompt()
+    {
+        // Arrange
+        var promptTemplateConfig = new PromptTemplateConfig("Hello AI")
+        {
+            Description = "My sample function."
+        };
+        promptTemplateConfig.InputVariables.Add(new InputVariable() { Name = "parameter1", Description = "String parameter", JsonSchema = "{\"type\":\"string\",\"description\":\"String parameter\"}" });
+        promptTemplateConfig.InputVariables.Add(new InputVariable() { Name = "parameter2", Description = "Enum parameter", JsonSchema = "{\"enum\":[\"Value1\",\"Value2\"],\"description\":\"Enum parameter\"}" });
```

Comment:
Improve readability.

```suggestion
var promptTemplateConfig = new PromptTemplateConfig("Hello AI")
{
    Description = "My sample function.",
    InputVariables = new List<InputVariable>
    {
        new InputVariable 
        { 
            Name = "parameter1", 
            Description = "String parameter", 
            JsonSchema = "{\"type\":\"string\",\"description\":\"String parameter\"}"
        },
        new InputVariable 
        { 
            Name = "parameter2", 
            Description = "Enum parameter", 
            JsonSchema = "{\"enum\":[\"Value1\",\"Value2\"],\"description\":\"Enum parameter\"}"
        }
    }
};
```